README.md

```diff
@@ -1 +1,39 @@
-# Cadastro-central
+# Central register
+
```

Comment:
Maybe add "About" section here, describing **succinctly** what is the central_register, what it does, etc.